.github/workflows/release_github.yaml

```diff
@@ -0,0 +1,20 @@
+# On each new commit to master, create/update a PR with release
+# automatically bumps version and creates changelog as per conventional commits
+name: Release Github
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        id: release
+        with:
+          token: ${{ secrets.REPO_GHA_PAT }}
+          release-type: node
```

Comment:
the product itself won't run with nodejs

Reply:
yeah but this is so that release please mechanism knows that it needs to look for `package.json` and `package-lock.json` to change release versions

Reply:
Please see https://github.com/google-github-actions/release-please-action#release-types-supported